backend/lib-shared-site-api/src/clients/s3_client.rs

```diff
@@ -118,6 +118,25 @@ impl S3Client {
         )
     }
 
+    pub fn presign_put_site_backup(
```

Comment:
Can we reuse the existing `presign_put` function? It seems like the only difference is we need to pass `self.backup_bucket` as a parameter.